webmails/roundcube/start.py

```diff
@@ -11,22 +11,41 @@
 os.environ["MAX_FILESIZE"] = str(int(int(os.environ.get("MESSAGE_SIZE_LIMIT")) * 0.66 / 1048576))
 
 db_flavor = os.environ.get("ROUNDCUBE_DB_FLAVOR", "sqlite")
+db_port = os.environ.get("ROUNDCUBE_DB_PORT")
```

Comment:
What about this?
```
db_host = os.environ.get("ROUNDCUBE_DB_HOST", "database")
if db_port := os.environ.get("ROUNDCUBE_DB_PORT"):
  db_host += f':{db_port}'
```
and instead of your changes replace `os.environ.get("ROUNDCUBE_DB_HOST", "database")` with `db_host` in the original code.